pkg/app/cmd.go

```diff
@@ -211,6 +211,7 @@ func (o *options) bindFlags(flags *pflag.FlagSet) {
 		defaultRunID = uuid.New().String()
 	}
 	flags.StringVar(&o.runid, "run-id", defaultRunID, "unique identifier for a kubetest2 run")
+	flags.BoolVar(&o.cpRundirToArtifacts, "cp-rundir-to-artifacts", false, `To decide whether the test binaries and run specific metadata have to be copied to the ARTIFACTS `)
```

Comment:
@BenTheElder I reframed by removing the additional copying.
`rundir` would be under ARTIFACTS Only when `--cp-rundir-to-artifacts` is true.

Did this by joining the RunDir to artifacts BaseDir when the copy flag is true.
Had to create a new function `subRunDir` to avoid rundir with absolute path when it is created under ARTIFACTS.

Please take a look.